src/main/kotlin/network/warzone/mars/player/achievements/models/Achievement.kt

```diff
@@ -0,0 +1,161 @@
+package network.warzone.api.database.models
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import network.warzone.mars.feature.NamedResource
+import network.warzone.mars.player.achievements.AchievementAgent
+import network.warzone.mars.player.achievements.models.AchievementParent
+import java.util.*
+
+enum class AgentType {
+    TOTAL_KILLS_AGENT, /** WORKS **/
+    TOTAL_DEATHS_AGENT, /** WORKS **/
+    TOTAL_WINS_AGENT, /** WORKS **/
+    TOTAL_LOSSES_AGENT, /** WORKS **/
+    KILL_STREAK_AGENT, /** WORKS **/
+    FIRE_DEATH_AGENT, /** WORKS **/
+    CHAT_MESSAGE_AGENT, /** WORKS **/
+    LEVEL_UP_AGENT, /** WORKS **/
+    CAPTURE_NO_SPRINT_AGENT, /** DOESN'T WORK **/
```

Comment:
Could you mention the ones that don't work somewhere in the PR description so it is more visible